modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -4042,6 +4047,10 @@ public void onDiscoveryEvent(
      * @return {@code True} if minor topology version should be increased.
      */
     public boolean onCustomEvent(DiscoveryCustomMessage msg, AffinityTopologyVersion topVer, ClusterNode node) {
+        if (msg instanceof InitMessage &&
```

Comment:
Should be removed if standard discovery workflow will be used.

Reply:
Removed, since the API of `DistributedProcess` changed.